docs/en/drivers/ios-xcuitest-real-devices.md

```diff
@@ -236,6 +236,7 @@ for this).
 1. Make sure UDID is correct by checking it in Xcode Organizer or iTunes. It
    is a long string (20+ chars).
 1. Make sure that you can run your tests against the Simulator.
-1. Make sure UI Automation is enabled on your device. Settings -> Developer ->
-   Enable UI Automation
+1. Make sure the following settings are **enabled** on your device:
+    1. Settings -> Developer -> **Enable UI Automation**
+    1. Settings -> Safari -> Advanced -> **Web Inspector** and **Remote Automation**
```

Comment:
Could you add `Please read [Automating mobile web apps](docs/en/writing-running-appium/web/mobile-web/) for more details about WebView` as the next line?

https://appium.io/docs/en/writing-running-appium/web/mobile-web/

Reply:
done!

Reply:
thanks :)